src/legacy/core_plugins/kibana/public/home/components/home.js

```diff
@@ -51,7 +51,7 @@ export class Home extends Component {
       getServices().getInjected('disableWelcomeScreen') ||
       props.localStorage.getItem(KEY_ENABLE_WELCOME) === 'false'
     );
-    const showTelemetryDisclaimer = getServices().getInjected('allowChangingOptInStatus');
+    const showTelemetryDisclaimer = getServices().getInjected('telemetryNotifyUserAboutOptInDefault');
```

Comment:
Can you breifly explain this variable? and why we're not using a combination of `allowChangingOptInStatus` and `telemetryNotifyUserAboutOptInDefault`?

Reply:
`telemetryNotifyUserAboutOptInDefault` includes `telemetryNotifyUserAboutOptInDefault` _plus_ our new Saved-Object field (to show/hide this disclaimer). This way we only show it one time when the user starts.